proto/regen/ecocredit/v1/state.proto

```diff
@@ -220,4 +223,8 @@ message BatchSupply {
     // cancelled, effectively undoing the issuance. The sum of total_amount and
     // amount_cancelled will always equal the original credit issuance amount.
     string cancelled_amount = 4;
+
+    // escrowed_amount is the amount of credits in the batch that have been
+    // locked up in escrow for use in the marketplace.
+    string escrowed_amount = 5;
```

Comment:
Do we really need to have a separate supply here? I guess maybe that's a useful invariant?

Reply:
it seemed right to have it as the credits aren't necessarily tradable nor retired. if anything, it might be useful to be able to query a batch to see how many credits are in the market?